webapp/src/shared/components/Input/SearchUserForm.tsx

```diff
@@ -0,0 +1,25 @@
+import { Input, InputVariant } from '../';
+import { IconVariant } from '../Icon/Icon';
+import './SearchUserForm.css';
+
+type SearchProps = {
+  search: string;
+  setSearch: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export default function SearchUserForm({ search, setSearch }: SearchProps) {
```

Comment:
Now this one is a form specific to search users, but since we are not hard coding here the endpoint we're using we may as well make it generic, isn't it?
Search form, like that we may use it in the future to search chats or games

Reply:
I've update it to searchForm in a more generic way